message_event.go

```diff
@@ -60,20 +60,20 @@ func (e *MessageEvent) IsThread() bool {
 
 // IsBot indicates if the message was sent by a bot
 func (e *MessageEvent) IsBot() bool {
-	return e.BotID != ""
```

Comment:
This seems like an unnecessary stylistic change and less clear imo.

`MessageEvent.BotID` is a string and it seems clearer to treat it as such, since we don't normally interact with them by index like we might a slice where checking length would be more appropriate.

At the end of the day they should both compile to the [same thing](https://godbolt.org/z/631ncd9ns).

Reply:
Interesting. I remember `len(stringValue) > 0` performing much much better than `stringValue != ""`

No worries, I can revert it